<?xml version="1.0" encoding="UTF-8"?>
<testng-results ignored="0" total="5" passed="4" failed="1" skipped="0">
  <reporter-output>
  </reporter-output>
  <suite started-at="2023-08-03T08:51:50Z" name="Suite" finished-at="2023-08-03T08:52:51Z" duration-ms="61288">
    <groups>
      <group name="Purchase">
        <method signature="StandAloneTest2.submitOrder(java.util.HashMap)[pri:0, instance:com.workpulse.test.StandAloneTest2@4879dfad]" name="submitOrder" class="com.workpulse.test.StandAloneTest2"/>
      </group> <!-- Purchase -->
      <group name="ErrorHandling">
        <method signature="ErrorValidationTest.loginErrorValidation()[pri:0, instance:com.workpulse.test.ErrorValidationTest@66565121]" name="loginErrorValidation" class="com.workpulse.test.ErrorValidationTest"/>
        <method signature="ErrorValidationTest.productErrorValidation()[pri:0, instance:com.workpulse.test.ErrorValidationTest@66565121]" name="productErrorValidation" class="com.workpulse.test.ErrorValidationTest"/>
      </group> <!-- ErrorHandling -->
    </groups>
    <test started-at="2023-08-03T08:51:50Z" name="Submit the Order" finished-at="2023-08-03T08:52:31Z" duration-ms="41153">
      <class name="com.workpulse.test.StandAloneTest2">
        <test-method is-config="true" signature="launchApplication()[pri:0, instance:com.workpulse.test.StandAloneTest2@4879dfad]" started-at="2023-08-03T08:51:50Z" name="launchApplication" finished-at="2023-08-03T08:51:55Z" duration-ms="5144" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- launchApplication -->
        <test-method signature="submitOrder(java.util.HashMap)[pri:0, instance:com.workpulse.test.StandAloneTest2@4879dfad]" started-at="2023-08-03T08:51:55Z" name="submitOrder" data-provider="getData" finished-at="2023-08-03T08:52:09Z" duration-ms="14496" status="PASS">
          <params>
            <param index="0">
              <value>
                <![CDATA[{password=Artis!@#123, countryName=india, email=rohankg09@gmail.com, productName=ZARA COAT 3}]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- submitOrder -->
        <test-method is-config="true" signature="tearDown()[pri:0, instance:com.workpulse.test.StandAloneTest2@4879dfad]" started-at="2023-08-03T08:52:09Z" name="tearDown" finished-at="2023-08-03T08:52:10Z" duration-ms="294" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDown -->
        <test-method is-config="true" signature="launchApplication()[pri:0, instance:com.workpulse.test.StandAloneTest2@4879dfad]" started-at="2023-08-03T08:52:10Z" name="launchApplication" finished-at="2023-08-03T08:52:13Z" duration-ms="3091" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- launchApplication -->
        <test-method signature="submitOrder(java.util.HashMap)[pri:0, instance:com.workpulse.test.StandAloneTest2@4879dfad]" started-at="2023-08-03T08:52:13Z" name="submitOrder" data-provider="getData" finished-at="2023-08-03T08:52:27Z" duration-ms="14196" status="PASS">
          <params>
            <param index="0">
              <value>
                <![CDATA[{password=Artis!@#123, countryName=india, email=rohankg09@gmail.com, productName=ADIDAS ORIGINAL}]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- submitOrder -->
        <test-method is-config="true" signature="tearDown()[pri:0, instance:com.workpulse.test.StandAloneTest2@4879dfad]" started-at="2023-08-03T08:52:27Z" name="tearDown" finished-at="2023-08-03T08:52:28Z" duration-ms="622" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDown -->
        <test-method is-config="true" signature="launchApplication()[pri:0, instance:com.workpulse.test.StandAloneTest2@4879dfad]" started-at="2023-08-03T08:52:28Z" name="launchApplication" finished-at="2023-08-03T08:52:30Z" duration-ms="2313" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- launchApplication -->
        <test-method signature="orderHistoryTest()[pri:0, instance:com.workpulse.test.StandAloneTest2@4879dfad]" started-at="2023-08-03T08:52:30Z" name="orderHistoryTest" finished-at="2023-08-03T08:52:31Z" depends-on-methods="com.workpulse.test.StandAloneTest2.submitOrder" duration-ms="654" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- orderHistoryTest -->
        <test-method is-config="true" signature="tearDown()[pri:0, instance:com.workpulse.test.StandAloneTest2@4879dfad]" started-at="2023-08-03T08:52:31Z" name="tearDown" finished-at="2023-08-03T08:52:31Z" duration-ms="149" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDown -->
      </class> <!-- com.workpulse.test.StandAloneTest2 -->
    </test> <!-- Submit the Order -->
    <test started-at="2023-08-03T08:52:31Z" name="Validate the Error Handeling" finished-at="2023-08-03T08:52:51Z" duration-ms="19822">
      <class name="com.workpulse.test.ErrorValidationTest">
        <test-method is-config="true" signature="launchApplication()[pri:0, instance:com.workpulse.test.ErrorValidationTest@66565121]" started-at="2023-08-03T08:52:31Z" name="launchApplication" finished-at="2023-08-03T08:52:33Z" duration-ms="2331" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- launchApplication -->
        <test-method signature="loginErrorValidation()[pri:0, instance:com.workpulse.test.ErrorValidationTest@66565121]" started-at="2023-08-03T08:52:33Z" name="loginErrorValidation" finished-at="2023-08-03T08:52:34Z" duration-ms="347" status="FAIL">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[expected [Incorrect email or password.] but found [Incorrect email  password.]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: expected [Incorrect email or password.] but found [Incorrect email  password.]
at org.testng.Assert.fail(Assert.java:96)
at org.testng.Assert.failNotEquals(Assert.java:776)
at org.testng.Assert.assertEqualsImpl(Assert.java:137)
at org.testng.Assert.assertEquals(Assert.java:118)
at org.testng.Assert.assertEquals(Assert.java:453)
at org.testng.Assert.assertEquals(Assert.java:463)
at com.workpulse.test.ErrorValidationTest.loginErrorValidation(ErrorValidationTest.java:24)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:568)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.Invoker.invokeMethod(Invoker.java:583)
at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:719)
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:989)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
at org.testng.TestRunner.privateRun(TestRunner.java:648)
at org.testng.TestRunner.run(TestRunner.java:505)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)
at org.testng.SuiteRunner.run(SuiteRunner.java:364)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1137)
at org.testng.TestNG.runSuites(TestNG.java:1049)
at org.testng.TestNG.run(TestNG.java:1017)
at org.apache.maven.surefire.testng.TestNGExecutor.run(TestNGExecutor.java:308)
at org.apache.maven.surefire.testng.TestNGXmlTestSuite.execute(TestNGXmlTestSuite.java:71)
at org.apache.maven.surefire.testng.TestNGProvider.invoke(TestNGProvider.java:113)
at org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:385)
at org.apache.maven.surefire.booter.ForkedBooter.execute(ForkedBooter.java:162)
at org.apache.maven.surefire.booter.ForkedBooter.run(ForkedBooter.java:507)
at org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:495)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- loginErrorValidation -->
        <test-method is-config="true" signature="tearDown()[pri:0, instance:com.workpulse.test.ErrorValidationTest@66565121]" started-at="2023-08-03T08:52:34Z" name="tearDown" finished-at="2023-08-03T08:52:34Z" duration-ms="119" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDown -->
        <test-method is-config="true" signature="launchApplication()[pri:0, instance:com.workpulse.test.ErrorValidationTest@66565121]" started-at="2023-08-03T08:52:34Z" name="launchApplication" finished-at="2023-08-03T08:52:37Z" duration-ms="3069" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- launchApplication -->
        <test-method signature="productErrorValidation()[pri:0, instance:com.workpulse.test.ErrorValidationTest@66565121]" started-at="2023-08-03T08:52:37Z" name="productErrorValidation" finished-at="2023-08-03T08:52:51Z" duration-ms="13533" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- productErrorValidation -->
        <test-method is-config="true" signature="tearDown()[pri:0, instance:com.workpulse.test.ErrorValidationTest@66565121]" started-at="2023-08-03T08:52:51Z" name="tearDown" finished-at="2023-08-03T08:52:51Z" duration-ms="200" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDown -->
      </class> <!-- com.workpulse.test.ErrorValidationTest -->
    </test> <!-- Validate the Error Handeling -->
  </suite> <!-- Suite -->
</testng-results>
